roles/firewall/tasks/main.yml

```diff
@@ -58,27 +62,43 @@
     loop_var: ip
   when: firewall.ipv6
 
-- name: Define inbound IPs
+- name: Define inbound IPs.
   set_fact:
     _firewall_inbound: "{{ firewall.inbound + firewall._inbound_defaults }}"
 
 - name: Define outbound IPs
   set_fact:
     _firewall_outbound: "{{ firewall.outbound + firewall._outbound_defaults }}"
 
-- name: Allow firewall inbound rules
+- name: Allow firewall inbound rules.
   include_tasks: inbound.yml
   with_items: "{{ _firewall_inbound }}"
   loop_control:
     loop_var: inbound
 
-- name: Allow firewall outbound rules
+- name: Allow firewall outbound rules.
   include_tasks: outbound.yml
   with_items: "{{ _firewall_outbound }}"
   loop_control:
     loop_var: outbound
 
-- name: Configure Additional Firewall Rules
+
+- name: Enable NAT.
+  blockinfile:
+    path: /etc/ufw/before.rules
+    block: |
+      # NAT table rules
+      *nat
+      :POSTROUTING ACCEPT [0:0]
+      # Forward traffic through eth0 - Change to match you out-interface
+      -A POSTROUTING -o {{ ansible_default_ipv4.interface }} -j MASQUERADE
+      # don't delete the 'COMMIT' line or these nat table rules won't
+      # be processed
+      COMMIT
+    insertbefore: "^# Don't delete these required lines"
+    when: firewall.nat
```

Comment:
Is the indentation right on `when: firewall.nat`? Should it be lined up with `name` and `blockinfile` instead?